s3Emulator/preload/loaddb.sh

```diff
@@ -6,8 +6,8 @@ chmod +x *
 
 ./create_analysis.sh || { echo "create_analysis.sh failed"; exit 1; }
 
-./create_systemmetadata.sh || { echo "create_systemmetadata.sh failed"; exit 1; }
+# ./create_systemmetadata.sh || { echo "create_systemmetadata.sh failed"; exit 1; }
 
-./create_file_metadata.sh || { echo "create_file_metadata.sh failed"; exit 1; }
+# ./create_file_metadata.sh || { echo "create_file_metadata.sh failed"; exit 1; }
 
-./create_validation.sh || { echo "create_file_metadata.sh failed"; exit 1; }
+# ./create_validation.sh || { echo "create_file_metadata.sh failed"; exit 1; }
```

Comment:
This branch has several randomly modified messy code.. We need to consider clean it a bit @jizhuoyu

Reply:
Ok, me and @wmao17 plan to prepare for demo of the complete submission workflow today (so that we can show the process of user submitting a request in frontend and submission worker pulling the submitted algorithm before invoking the runner which will produce a fake output for now). After the demo is working we can consider to clean the repo?